src/test/resources/route-descriptions.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "source": {
+      "topic": {
+        "name": "bahmni-patient"
+      }
+    },
+    "destinations": [
+      {
+        "onEventType": "BAHMNI_PATIENT_CREATED",
+        "topic": {
+          "name": "patient-registration"
+        }
+      }
+    ],
+    "errorDestination": {
+      "queue": {
+        "name": "event-router-service:bahmni-patient-dlq"
+      },
+      "maxRetryDelivery": 2,
+      "retryDeliveryDelayInMills": 2000,
+      "cronExpressionForRetryStart": "* 5 * * * ?",
+      "cronExpressionForRetryStop": "* 7 * * * ?"
+    },
+    "additionalProperties": {
+      "facility": "Ethopia"
+    },
+    "filterOnProperties": {
```

Comment:
I would like JSON configuration to be like this: 
- source { topicName }, 
- eventsToRead { eventType, filterOnProps, description}, 
- destination (onSuccess {}, onError {}, enrichEvent {additionalProperties}, description)

This way, it is clear that there are 3 things happening. Source, EventFiltering, DestinationWriting
Also, a description attribute that can act as documentation to say what I am trying to achieve in this section.

If this change is easy, please make it. If too difficult. Then let it be. thanks! 
In future, when deciding on a configuration, please have it reviewed BEFORE writing the code. The configuration is the interface for customers, and it needs to be intuitive. They won't see the code. But they will definitely see the configuration.

Reply:
No worries. Unfortunately, once the config is used in production, changing it will mean an "upgrade" migration step. : ) Ignore this for now. Don't create a backlog card.